src/test/scala/caio/CaioTests.scala

```diff
@@ -142,6 +147,28 @@ class CaioTests extends AsyncFunSpec with Matchers {
     }
   }
 
+  describe("Concurrency") {
+    implicit val CS: ContextShift[IO]    = IO.contextShift(ExecutionContext.global)
+    implicit val ioTimer: Timer[IO]      = IO.timer(ExecutionContext.global)
+    implicit val caioTimer: Timer[CaioT] = new Timer[CaioT] {
+      val clock: Clock[CaioT]                          = Clock.create[CaioT]
+      def sleep(duration: FiniteDuration): CaioT[Unit] = staticCaioConcurrent.liftIO(ioTimer.sleep(duration))
+    }
+
+    it("Should handle computations in parallel") {
```

Comment:
Some more tests are required here:
Failures, do they append?
Events, do they combine?
Exeptions - dunnot what to expect? How does IO do it?

Reply:
More tests added.

- Failures won't append. The second fiber is cancelled as soon as the first one fails.
- Same case with errors which is exactly how Cats IO behaves.
- Events combine.
- As we already discussed the last context joined will be taken.